frontend/src/scenes/notebooks/Notebook/Notebook.tsx

```diff
@@ -37,11 +37,8 @@ export function Notebook({ shortId, editable = false, initialAutofocus = null }:
     }, [])
 
     useEffect(() => {
-        if (editor) {
-            editor.setEditable(editable)
-            editor.focus(initialAutofocus)
```

Comment:
Removing this will break the behaviour where we set the focus to the replay timestamp when creating a new notebook from the 'Comment' button on a recording

Reply:
If you expand the code, its in the useEffect below? Does it need to run twice??

Reply:
It still works so I assumed it was just a copy paste issue

Reply:
Ah, my bad. Missed that